OpenTabletDriver.Daemon.Library/Extensions.cs

```diff
@@ -15,7 +14,7 @@ public static class Extensions
             if (path == null)
                 return null;
 
-            var type = pluginFactory.GetPluginType(path);
+            var type = pluginFactory.GetPlugin(path);
             var attr = type?.GetCustomAttribute<PluginNameAttribute>();
             return attr != null ? attr.Name : type?.Name;
```

Comment:
variable naming nitpick:
```suggestion
            var plugin = pluginFactory.GetPlugin(path);
            var attr = plugin?.GetCustomAttribute<PluginNameAttribute>();
            return attr != null ? attr.Name : plugin?.Name;
```

Reply:
It's not an instance of a plugin yet, so calling it `plugin` feels a bit weird to me. `pluginType` then?

Reply:
> It's not an instance of a plugin yet, so calling it `plugin` feels a bit weird to me. `pluginType` then?

The logic is mostly based off the function rename, but `type` is also ambiguous, so I agree with `pluginType`

Reply:
Resolved